GOVERNANCE.md

```diff
@@ -244,7 +277,153 @@ The current NumFOCUS Subcommittee consists of:
 - Jon Sedar
 - Thomas Wiecki
 
-### Institutional Partners and Funding
+## Core Contributors
+Core Contributors are those individuals with commit or managing rights to any of
+the repositories or services of the PyMC Project. However, the exact permissions
+of all Core Contributors may not be the same and depend on their team memberships.
+Therefore, Core Contributors may act as Recurrent Contributors depending
+on the task at hand.
```

Comment:
That sounds like the "core contributors" are not a GitHub Team?
> Core Contributors are those individuals with commit or managing rights to **any** of
the repositories

Phrased like this, any PyMC3 or Docs Team member is automatically a core contributor.

> Therefore, Core Contributors may act as Recurrent Contributors depending
on the task at hand.

I'm not sure what this means

Reply:
> That sounds like the "core contributors" are not a GitHub Team?

No, I have tried to make everything less code centered and less GitHub centered. In my opinion, we could potentially have core contributors or council members who don't even have a GitHub account if they were for example Discourse admin or moderator.

> Phrased like this, any PyMC3 or Docs Team member is automatically a core contributor.

We probably need to fix the wording. What I was trying to phrase is that someone with write permissions to pymc-examples is a core contributor, even if they don't have write permissions on pymc3 repo, or like I said above, no write permissions on github but moderator on discourse. Members of dev or doc team don't necessarily have write permissions to any repo, teams are comprised of core and recurrent contributors, only core contributors would have write permission.

> I'm not sure what this means

This links to the previous point, and is mostly being sensible but I included that in hopes of being transparent and explicit. An example of that would be that while someone might be a core contributor with write permissions to pymc3 (part of dev team), they should not merge a doc only PR without some approval from a doc team member (depending on the extent and complexity of the PR)

Reply:
> Core Contributors are those individuals with commit or managing rights

If you want to explicitly disentangle Core Contributor membership from specific services (code/docs, etc.) then I would not start with this.